### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by IvanTymoshchuk** [IvanTymoshchuk/test-work#1] [Q&A]
Оголоси наступні змінні, використовуючи ключове слово const або let, і присвой їм відповідні значення.

topSpeed - число 160.
distance - число 617.54.
login - рядок "mango935".
isOnline - буль true.
isAdmin - буль false.

**Comment by ViktorSvertoka** [comment]
const topSpeed = 160;
  const distance = 617.54;
  const login = "mango935";
  const isOnline = true;
  const isAdmin = false;
